app/CocoaPods/CPSplashRecentDocumentsController.m

```diff
@@ -0,0 +1,86 @@
+//
+//  CPSplashRecentDocumentsController.m
+//  CocoaPods
+//
+//  Created by Orta Therox on 05/09/2015.
+//  Copyright (c) 2015 CocoaPods. All rights reserved.
+//
+
+#import "CPSplashRecentDocumentsController.h"
+
+
+@interface _CPSplashDocument : NSObject <NSCopying>
+@property (nonatomic, copy) NSString *name;
+@property (nonatomic, copy) NSURL *podfileURL;
+@property (nonatomic, copy) NSImage *image;
+@property (nonatomic, copy) NSString *folderPath;
+@end
+
+@implementation _CPSplashDocument
+
+// Needed for using Cocoa Bindings
+
+- (instancetype)copyWithZone:(NSZone *)zone
+{
+  _CPSplashDocument *copy = [[[self class] alloc] init];
+  if (copy) {
+    [copy setName:[self.name copyWithZone:zone]];
+    [copy setPodfileURL:[self.podfileURL copyWithZone:zone]];
+    [copy setImage:[self.image copyWithZone:zone]];
+    [copy setFolderPath:[self.folderPath copyWithZone:zone]];
+  }
+  
+  return copy;
+}
+
```

Comment:
I think we should implement a `description` method which returns the Podfile.

So we don't expose the class/pointer to the user but instead a more useful string:

![screen shot 2015-09-08 at 11 16 03](https://cloud.githubusercontent.com/assets/44164/9743400/088f9e76-561b-11e5-8513-31958b1508dc.png)